components/yaml/introduction.rst

```diff
@@ -250,6 +233,77 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Handling Invalid Types
+......................
+
+By default the parser will encode invalid types as "null". You can make the
+parser throw exceptions by using the ``PARSE_EXCEPTION_ON_INVALID_TYPE``
+flag::
+
+    $yaml = '!php/object:O:8:"stdClass":1:{s:5:"foo";s:7:"bar";}';
+    Yaml::parse($yaml, Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE); // throws an exception
+
+Similarly you can use ``DUMP_EXCEPTION_ON_INVALID_TYPE`` when dumping::
+
+    $data = new \stdClass(); // by default objects are invalid.
+    Yaml::parse($data, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE); // throws an exception
+
+Using Objects for Maps
+......................
+
+.. versionadded:: 2.7
+    Support for parsing mappings as objects was introduced in Symfony 2.7.
```

Comment:
versionadded can be removed (all 2.x versionaddeds were removed from the 3.0 docs)